api-server/src/server/utils/getSetAccessToken.js

```diff
@@ -20,7 +20,9 @@ export function setAccessTokenToResponse(
 ) {
   const cookieConfig = {
     ...createCookieConfig(req),
-    maxAge: accessToken.ttl || 77760000000
+    maxAge: accessToken.ttl || 77760000000,
+    httpOnly: true,
+    secure: process.env.FREECODECAMP_NODE_ENV === 'production'
```

Comment:
```suggestion
    secure: process.env.FREECODECAMP_NODE_ENV !== 'development'
```
Not a big deal, but I think it's preferable to default to secure and make exceptions for specific cases.

Reply:
You are supposed to be reviewing the client-side...

Reply:
Ooops. I did that, too, and everything is fine there.